sources/web/datalab/polymer/components/item-list/item-list.ts

```diff
@@ -54,6 +57,16 @@ class ItemListElement extends Polymer.Element {
    */
   public columns: Array<string>;
 
+  /**
+   * Whether to hide the header row
+   */
+  public noHeader: boolean;
```

Comment:
This feels like it should be the default behavior though. If you're adding an `item-list` element, you should explicitly tell it if you want to disable that feature, rather than add it.